ext/webgpu/byow.rs

```diff
@@ -161,6 +161,25 @@ impl UnsafeWindowSurface {
 
     context.present().map_err(JsErrorBox::from_err)
   }
+
+  #[fast]
+  fn resize(
+    &self,
+    width: u32,
+    height: u32,
+    scope: &mut v8::HandleScope,
+  ) -> Result<(), JsErrorBox> {
+    self.width.replace(width);
+    self.height.replace(height);
+
+    let Some(context) = self.context.try_unwrap(scope) else {
+      return Ok(());
+    };
+
+    context
+      .resize_configure(width, height)
+      .map_err(JsErrorBox::from_err)
```

Comment:
```suggestion
    context.resize_configure(width, height)
```
unecessary map_err since resize_configure returns a JsErrorBox

Reply:
Fixed, now they do not return `Result`s.